docs/en/migration/model.md

```diff
@@ -1,3 +1,477 @@
 # Migrate Model from MMCV to MMEngine
 
-Coming soon. Please refer to [chinese documentation](https://mmengine.readthedocs.io/zh_CN/latest/migration/model.html).
+## Introduction
+
+The early computer vision tasks supported by MMCV, such as detection and classification, used a general process to optimize model. It can be summarized as the following four steps:
+
+1. Calculate the loss
+2. Calculate the gradients
+3. Update the model parameters
+4. Clean the gradients of the last iteration
+
+For most of high-level tasks, "where" and "when" to perform the above processes is commonly fixed, therefore it seems reasonable to use [Hook](../design/hook.md) to implement it. MMCV implements series of hooks, such as `OptimizerHook`, `Fp16OptimizerHook` and `GradientCumulativeFp16OptimizerHook` to provide varies of optimization strategy.
+
+On the other hand, tasks like GAN(Generative adversarial network) and Self-supervision require more flexible training processes, which does not meet the characteristics mentioned above, and it could be hard to use hooks to implement them. To meet the needs of these of tasks, MMCV will pass `optimizer` to `train_step` and users can customize the optimization process as they want. Although it works, it cannot utilize varies of `OptimizerHook` implemented in MMCV, and downstream repositories have to implement mix-precision training, gradient accumulative by their own.
```

Comment:
```suggestion
On the other hand, tasks like GAN(Generative adversarial network) and Self-supervision require more flexible training processes, which does not meet the characteristics mentioned above, and it could be hard to use hooks to implement them. To meet the needs of these tasks, MMCV will pass `optimizer` to `train_step` and users can customize the optimization process as they want. Although it works, it cannot utilize varies of `OptimizerHook` implemented in MMCV, and downstream repositories have to implement mix-precision training, gradient accumulation on their own.
```